.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -2034,12 +2044,38 @@ stages:
             artifact: linux-packages-arm64
             path: $(Build.ArtifactStagingDirectory)
 
+        # Linux symbols
+        - task: DownloadPipelineArtifact@2
+          displayName: Download linux Alpine symbols
+          inputs:
+            artifact: linux-symbols-alpine
+            path: $(Build.ArtifactStagingDirectory)/symbols/linux-musl-x64
+  
+        - task: DownloadPipelineArtifact@2
+          displayName: Download linux Debian symbols
+          inputs:
+            artifact: linux-symbols-debian
+            path: $(Build.ArtifactStagingDirectory)/symbols/linux-x64
+  
+        - task: DownloadPipelineArtifact@2
+          displayName: Download linux Arm64 symbols
+          inputs:
+            artifact: linux-symbols-arm64
+            path: $(Build.ArtifactStagingDirectory)/symbols/linux-arm64
+
+        - bash: |
+            tar -xcf  $(Build.ArtifactStagingDirectory)/linux-native-symbols.tar.gz $(Build.ArtifactStagingDirectory)/symbols
```

Comment:
Not sure we actually need [that section of the release doc](https://datadoghq.atlassian.net/wiki/spaces/APMINT/pages/2258338076/Release+Process#Release-steps) anymore (the one asking us to check the uploaded files).
So please update or remove the step entirely.